### DISCUSSION TITLE
How do I configure adding fields to each table？

### DISCUSSION THREAD

**Discussion by MrFriedRice** [sequelize/sequelize#16342] [Help & Questions]
![image](https://github.com/sequelize/sequelize/assets/40545975/76caf713-343e-4577-8eb3-566c6ebb1cbf)
Do this once for almost every table 
 new Sequelize({})
 
 Can configure it here?

**Comment by ephys** [comment]
You can use model inheritance for this, by adding these properties to a parent class

**Reply by MrFriedRice** [reply]
thank you~
